src/Raygun4php/RaygunClient.php

```diff
@@ -473,7 +473,7 @@ private function post($data_to_send, $cert_path)
     }
 
     function toJsonRemoveUnicodeSequences($struct) {
-      return preg_replace_callback("/\\\\u([a-f0-9]{4})/", function($matches){ return iconv('UCS-4LE','UTF-8',pack('V', hexdec("U$matches[1]"))); }, json_encode($struct));
+      return preg_replace_callback("/\\\\u([a-f0-9]{4})/", function($matches){ return iconv('UCS-4LE','UTF-8',pack('V', hexdec($matches[1]))); }, json_encode($struct));
```

Comment:
Did you consider copying the change from #126 directly into this version?

Reply:
I saw that, but I think the change is trying to solve the wrong problem.

I was going to raise a pull request against it in the 2.x line as well, but that code is no longer in 2.x, or at least not in that class anymore.